test/hotspot/jtreg/gc/shenandoah/generational/TestOldGrowthTriggers.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/*
+ * @test
+ * @summary Test that growth of old-gen triggers old-gen marking
+ * @requires vm.gc.Shenandoah
+ * @library /test/lib
+ * @run driver TestOldGrowthTriggers
+ */
+
+import java.util.*;
+import java.math.BigInteger;
+
+import jdk.test.lib.Asserts;
+import jdk.test.lib.process.ProcessTools;
+import jdk.test.lib.process.OutputAnalyzer;
+
+public class TestOldGrowthTriggers {
+
+  public static void do_old_allocations() {
```

Comment:
Nit, but Java convention is usually camel case: `doOldAllocations` or `makeOldAllocations` if the `oO` looks weird to you ;)

Reply:
Thanks.  Fixed.